### ISSUE TITLE
Job Applications area has disappeared from wordpress admin 

### ISSUE THREAD

**Issue by boostonline** [highrisedigital/wpbroadbean#52]
I'm not sure known to set the application page and fields within that.
Has this been moved to a paid add on?

**Comment by wpmark** [comment]
With the new version (v3) of the plugin, the settings have changed slightly. Users now get the choice of whether to use the application email or URL. If email us chosen then an application form is rendered beneath each job on the single job permalink page view. If a URL is chosen a button is displayed which links to that URL.

**Comment by boostonline** [comment]
Thanks.

I've made two-thirds progress. The application form is where I am stuck upgrading the old site. It was useful to see an application was submitted in the WordPress admin.

In a custom theme page template, the_content gets passed the from an application form function.
Past developers did some hardcoded changes for fields. Think I may have to do the same. I've not been able to find the plugin documentation to help me either.

Part of the old file
`/* get the apply page from the settings */
$apply_pageid = get_option( 'wpbb_apply_page_id' );

/* check this is the apply page */
if( ! is_page( $apply_pageid ) )
	return $content;
	/* check we have a job id passed */
	if( ! empty( $_GET[ 'job_id' ] ) ) {

		/* get the post for this job reference */
		$job_post = wpbb_get_job_by_reference( $_GET[ 'job_id' ] );

		/* check if we have a job post for this reference */
		if( $job_post == false ) {

			/* create an error message as no job has this reference */
			return '<p class="message error">Error: No job exists with this job reference.</p>' . $content;

		}
`

This is the only way I got something to show

I'm calling a function from this file: wpbroadbean/inc/applications/application-form.php

`add_filter( 'the_content', 'wpbb_application_form' );

function wpbb_application_form( $content ) {
	return wpbb_get_job_application_form( $_GET[ 'job_id' ] );

}`

Believe I have to modify application-form.php now to match how original devs did it.
